lib/attributor/types/model.rb

```diff
@@ -32,6 +32,7 @@ def self.inherited(klass)
 
       v = self.value_type
       va = self.value_attribute
+      re = self.requirements
```

Comment:
ya, I would treat it like the other things we have there now (value_type, value_attribute...etc)
if there's more complex inheritance needs we can revise later. I think that simple=better for now